ingest.py

```diff
@@ -81,32 +81,31 @@ def load(self) -> List[Document]:
 }
 
 
-def load_single_document(file_path: str) -> Document:
+def load_single_document(file_path: str) -> List[Document]:
     ext = "." + file_path.rsplit(".", 1)[-1]
     if ext in LOADER_MAPPING:
         loader_class, loader_args = LOADER_MAPPING[ext]
         loader = loader_class(file_path, **loader_args)
-        return loader.load()[0]
+        return loader.load()
 
     raise ValueError(f"Unsupported file extension '{ext}'")
 
-
 def load_documents(source_dir: str, ignored_files: List[str] = []) -> List[Document]:
     """
     Loads all documents from the source documents directory, ignoring specified files
     """
     all_files = []
     for ext in LOADER_MAPPING:
         all_files.extend(
-            glob.glob(os.path.join(source_dir, f"**/*{ext}"), recursive=True)
+        glob.glob(os.path.join(source_dir, f"**/*{ext}"), recursive=True)
```

Comment:
The formatting on this line seems to have been edited by accident

```suggestion
            glob.glob(os.path.join(source_dir, f"**/*{ext}"), recursive=True)
```